README.md

```diff
@@ -1,7 +1,7 @@
-# Eclipse Project for Concurrency Utilities
-Concurrency Utilities for Jakarta EE provides a simple, standardized API for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
+# Jakarta Concurrency
+Jakarta Concurrency provides a simple, standardized API for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
 
-This project provides the standard api for concurrency utilities. The EE4J project that implements this api is also on [GitHub](https://github.com/eclipse-ee4j/concurrency-ri).
+This project provides the standard api for jakarta concurrency. The EE4J project that implements this api is also on [GitHub](https://github.com/eclipse-ee4j/concurrency-ri).
```

Comment:
As a proper noun, "jakarta concurrency" should be capitalized as "Jakarta Concurrency" here.
The acronym, "api", should also be capitalized as "API" in these sentences, as it appears in the prior paragraph.